rs/messaging/src/message_routing/tests.rs

```diff
@@ -1646,16 +1646,12 @@ fn check_critical_error_counter_is_not_incremented_for_transient_error() {
 /// Get protobuf-encoded snapshot of the mainnet registry state (around jan. 2022)
 fn get_mainnet_delta_00_6d_c1() -> (TempDir, LocalStoreImpl) {
     let tempdir = TempDir::new().unwrap();
-    let store = LocalStoreImpl::new(tempdir.path());
     let changelog =
         compact_delta_to_changelog(ic_registry_local_store_artifacts::MAINNET_DELTA_00_6D_C1)
             .expect("")
             .1;
 
-    for (v, changelog_entry) in changelog.into_iter().enumerate() {
-        let v = RegistryVersion::from((v + 1) as u64);
-        store.store(v, changelog_entry).unwrap();
-    }
+    let store = LocalStoreImpl::from_changelog(changelog, tempdir.path()).unwrap();
     (tempdir, store)
 }
```

Comment:
This exact code is replicated 3x (here, under `/rs/p2p` and under `/rs/registry/local_registry`, but I couldn't decide on a good `test_utilities` crate to place it under.

Reply:
Moved.